### ISSUE TITLE
It does not work in Accessibility Mode.

### ISSUE THREAD

**Issue by rcdey** [mikaoj/BSImagePicker#85]
First of all, thanks for the nice tool. I am facing an issue while implementing in Accessibility Mode. It does not select anything. Also focus never goes to BSImagePicker.

**Comment by mikaoj** [comment]
There is a PR with accessibility fixes.
I'm on vacation right now, without my computer, so haven't had time to look at it yet :)

I’m Joakim Gyllström and I approve this message

På 2 augusti 2016 till 08:30:38, rcdey (notifications@github.com<mailto:notifications@github.com>) skrev:

First of all, thanks for the nice tool. I am facing an issue while implementing in Accessibility Mode. It does not select anything. Also focus never goes to BSImagePicker.

**Comment by rcdey** [comment]
Thanks Joakim. Should I take PR?
Enjoy your vacation.

**Comment by mikaoj** [comment]
Go ahead and review the PR if you want. And check out if it does what you want 👍 
(if that was what you were asking :))

**Comment by mikaoj** [comment]
Since the pull request was merged....I'm assuming this issue can be closed?
